Readme.md

```diff
@@ -7,7 +7,7 @@ For LabVIEW NXG versions see the [previous releases](https://github.com/ni/webvi
 
 The following examples demonstrate different ways of using WebVIs. Each directory contains example code and a `README.md` file explaining the example.
 
```

Comment:
The updated WebSockets gif looks good. Can you update the example itself to use the new example URL? Push the webvi changes (but not Build changes, those should be a follow-up PR).